jmeter/plans/apply.rb

```diff
@@ -10,16 +10,22 @@ def think(milliseconds = 500)
   think_time milliseconds
 end
 
-# 975 concurrent users, 15 minute run time
 test do
   cookies clear_each_iteration: true
   view_results_tree
   thread_count = 975
+  csv_data_set_config filename: 'jmeter-courses.csv'
 
-  threads count: thread_count, continue_forever: true, duration: 900 do
+  threads count: thread_count, continue_forever: true, duration: 3600 do
     #-> Sign up
     visit name: 'Account page', url: url('/candidate/account') do
       extract name: 'authenticity_token', regex: 'name="authenticity_token" value="(.+?)"'
+
+      jsr223_preprocessor(
+        name: 'Candidate UUID',
+        script: 'vars.put("candidate_uuid", UUID.randomUUID().toString())',
+        language: 'groovy',
```

Comment:
I like the idea of using groovy + data imported from a CSV file but I think we'll need to make changes to the Dockerfile for this to work. I have checked out your branch and the apply test plan doesn't work anymore. I suspect we'll need to add a groovy .jar to the mix and we'll also need to make sure the courses csv is available to jmeter within the container.